drivers/srmetadata.py

```diff
@@ -634,10 +634,10 @@ def getMetadataToWrite(self, sr_info, vdi_info, lower, upper, update_map, \
 
     # specific functions, to be implement by the child classes
     def getVdiInfo(self, Dict, generateSector=0):
-        return
+        return ''
```

Comment:
don't we want to `raise NotImplementedError()` here ?

Reply:
No as the return from this is appended to and it is overriden by implementation which has information to add here.